osidb/models.py

```diff
@@ -1487,6 +1489,139 @@ def bzsync(self, *args, bz_api_key, **kwargs):
         self.flaw.save(*args, bz_api_key=bz_api_key, **kwargs)
 
 
+class CVSSManager(
+    ACLMixinManager,
+    TrackingMixinManager,
+):
+    """CVSS manager"""
+
+    @staticmethod
+    def create_cvss(flaw, affect, vector, issuer, version, **extra_fields):
```

Comment:
Based on my comment to the vector bellow I think this should actually be rather maybe `create_or_update` (the `_cvss` part in the name is unnecessary as this is all CVSS) and by used to potentially update the vector.

Reply:
I am not sure what are the `extra_fields` but if this is eventually used for the update too we should account for them appropriately whatever it means.